doc/user/content/sql/create-connection.md

```diff
@@ -8,19 +8,123 @@ menu:
+customer to create a connection to your AWS PrivateLink service.
+{{< /warning >}}
+
+### Accepting connection requests
+
+If your AWS PrivateLink service is configured to to require acceptance of
+connection requests, you must manually approve the connection request from
+Materialize after executing `CREATE CONNECTION`. See [Accept or reject
+connection requests](https://docs.aws.amazon.com/vpc/latest/privatelink/configure-endpoint-service.html#accept-reject-connection-requests) in the
+AWS PrivateLink documentation.
+
+### Example
+
+```sql
+CREATE CONNECTION privatelink_svc TO AWS PRIVATELINK (
+    SERVICE NAME 'com.amazonaws.vpce.us-east-1.vpce-svc-0e123abc123198abc',
+    AVAILABILITY ZONES ('use1-az1', 'use1-az4')
+)
+```
+
+## Confluent Schema Registry
+
+A Confluent Schema Registry connection establishes a link to a [Confluent Schema
+Registry] server.
+
+You can use Confluent Schema Registry connections in the [`FORMAT`] clause of
+[`CREATE SOURCE`] and [`CREATE SINK`].
+
+Field                       | Value            | Required | Description
+----------------------------|------------------|:--------:| ------------
+`URL`                       | `text`           | ✓        | The schema registry URL.
+`SSL CERTIFICATE AUTHORITY` | secret or `text` |          | The absolute path to the certificate authority (CA) certificate in PEM format. Used for both SSL client and server authentication. If unspecified, uses the system's default CA certificates.
+`SSL CERTIFICATE`           | secret or `text` | ✓        | Your SSL certificate in PEM format. Required for SSL client authentication.
+`SSL KEY`                   | secret           | ✓        | Your SSL certificate's key in PEM format. Required for SSL client authentication.
+`PASSWORD`                  | secret           |          | The password used to connect to the schema registry with basic HTTP authentication. This is compatible with the `ssl` options, which control the transport between Materialize and the CSR.
+`USERNAME`                  | secret or `text` |          | The username used to connect to the schema registry with basic HTTP authentication. This is compatible with the `ssl` options, which control the transport between Materialize and the CSR.
+`AWS PRIVATELINK`           | object name      |          | The name of an [AWS PrivateLink connection](#aws-privatelink) through which network traffic should be routed.
+
+### Example
+
+```sql
+CREATE SECRET csr_ssl_crt AS '<CSR_SSL_CRT>';
+CREATE SECRET csr_ssl_key AS '<CSR_SSL_KEY>';
+CREATE SECRET csr_password AS '<CSR_PASSWORD>';
+
+CREATE CONNECTION csr_ssl TO CONFLUENT SCHEMA REGISTRY (
+    URL 'https://rp-f00000bar.data.vectorized.cloud:30993',
+    SSL KEY = SECRET csr_ssl_key,
+    SSL CERTIFICATE = SECRET csr_ssl_crt,
+    USERNAME = 'foo',
+    PASSWORD = SECRET csr_password
+);
+```
+
 ## Kafka
 
+A Kafka connection establishes a link to a [Kafka] cluster.
```

Comment:
Does any validation happen when `CREATE CONNECTION` is run? As in, is this "describing how to connect", or is this "establishing a link"? If we only really connect on `CREATE SOURCE`, I would phrase it differently here. The current blurb for `CREATE SOURCE` is:

> [CREATE SOURCE](https://materialize.com/docs/sql/create-source/) connects Materialize to an external system you want to read data from, and provides details about how to decode and interpret that data.